scripts/update-vscode-dependencies.ts

```diff
@@ -34,11 +44,28 @@ async function updateVSCodeDependencies () {
       apiPackageJson.overrides[dependency] = vsCodePackageJson.dependencies[dependency]
     }
   }
+
+  console.debug('Updating monaco-vscode-api ncurc to prevent VSCode dependencies from being updated by ncu')
+  const ncuRcPath = path.join(__dirname, '../.ncurc.json')
+  const ncuRc = await readJson(ncuRcPath)
+  ncuRc.reject = [
+    '@types/node',
```

Comment:
Aren't those already contained in `.ncurc.json`?

Reply:
Yes, and I want to keep them! it I don't put theme here, they'll be removed